packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -90,6 +146,94 @@ export namespace AgenticaStart {
       services,
     });
 
-    console.log(`🎉 Project ${projectName} created`);
+    console.log(`\n🎉 Project ${projectName} created`);
+
+    console.log(
+      `\n⚠️  ${chalk.yellow("Note:")} Please implement constructor values for each controller generated in agent.ts or index.ts`,
+    );
   }
+
+  /**
+   * Get questions for `start` command.
+   */
+  const getQuestions = (
+    input: IAgenticaStart.IGetQuestionsInput,
+    options: IAgenticaStart.IOptions,
```

Comment:
You declared IGetQuestionsInput, but is there a specific reason for adding an additional positional parameter?
As far as I know, this team primarily focuses on using keyworded parameters